roxctl/utils/formatting_writer.go

```diff
@@ -73,44 +61,58 @@ func (w *formattingWriter) SetIndent(indent ...int) {
 // indentation and wrapping.
 // Implements the StringWriter interface.
 func (w *formattingWriter) WriteString(s string) (int, error) {
-	w.written = 0
-	tokenScanner := bufio.NewScanner(strings.NewReader(s))
-	tokenScanner.Split(wordsAndDelimeters)
-	var err error
-	for err == nil && tokenScanner.Scan() {
-		token := tokenScanner.Text()
-		length := len(token)
-		switch token {
-		case "\t":
-			length = defaultTabWidth
-		case "\n":
-			if w.currentLine == 0 {
-				w.indent.popNotLast()
-				w.indentReset = false
-			}
-			err = w.ln()
-			continue
+	written := 0
+	for token := range w.tokens(s) {
+		n, err := w.raw.WriteString(token)
+		written += n
+		if err != nil {
+			return written, err //nolint:wrapcheck
```

Comment:
i think it's better to wrap then disable this lintere. It takes almost the same number of keystrokes

Reply:
There's no much of useful information to add to the error, hence no wrapping.

Reply:
wrapping will add a trace